common/environment/environment.cpp

```diff
@@ -1920,6 +2019,31 @@ IConstDropZoneInfoIterator * CLocalEnvironment::getDropZoneIterator() const
     return new CConstDropZoneInfoIterator();
 }
 
+
+IConstDfuQueueInfoIterator * CLocalEnvironment::getDfuQueueIterator() const
+{
+    return new CConstDfuQueueInfoIterator();
+}
+
+bool CLocalEnvironment::isValidDfuQueueName(const char * queueName) const
+{
+    bool retVal = false;
+    if (queueName && *queueName)
+    {
+        Owned<IConstDfuQueueInfoIterator> queueIt = getDfuQueueIterator();
+        ForEach(*queueIt)
+        {
+            SCMStringBuffer _queueName;
+            queueIt->query().getDfuQueueName(_queueName);
+
+            LOG(MCdebugInfo, unknownJob, "Queue from cache: '%s'", _queueName.str());
```

Comment:
@AttilaVamos - verbose logging still in , see other previous comments re. logging

Reply:
Removed